R/is_lint_level.R

```diff
@@ -24,3 +24,23 @@ is_lint_level <- function(source_expression, level = c("expression", "file")) {
   required_key <- paste0(if (level == "file") "full_", "parsed_content")
```

Comment:
Should `is_lint_level()` be deprecated?

Reply:
Not at the moment. We need it for linters that run on both levels.

Reply:
But it could be sped up by avoiding a `paste0()` call.